src/TenancyServiceProvider.php

```diff
@@ -64,10 +65,13 @@ public function register(): void
         $this->app->singleton(Commands\Rollback::class, function ($app) {
             return new Commands\Rollback($app['migrator']);
         });
+
         $this->app->singleton(Commands\Seed::class, function ($app) {
             return new Commands\Seed($app['db']);
         });
 
+        $this->app->singleton(BaseVite::class, Vite::class);
```

Comment:
Sounds good to me, to move it to a optional feature so it can be disabled.

I'll mark this PR as draft and update it later today to convert this as an optional feature